site/gatsby-site/src/components/forms/SubmissionWizard/StepOne.js

```diff
@@ -142,6 +144,12 @@ const FormDetails = ({
     }
   }, [submissionFailed, submissionComplete]);
 
+  useEffect(() => {
```

Comment:
Does this trigger a fetch if you change the URL manually?

Reply:
Fixed